README.md

```diff
@@ -1,111 +1,191 @@
+- accountingbook
+- consolidatedexchangerate
+- currency
+- customer
+- classification
+- department
+- item
+- job
+- location
+- locationmainaddress
+- transactionaccountingline
+- transactionline
+- transaction
+- subsidiary
+- vendor
+- vendorcategory
+
+### Database Compatibility
+This package is compatible with either a **BigQuery**, **Snowflake**, **Redshift**, or **PostgreSQL** destination.
+
+## Step 2: Install the package
+Include the following netsuite package version in your `packages.yml` file:
+> TIP: Check [dbt Hub](https://hub.getdbt.com/) for the latest installation instructions or [read the dbt docs](https://docs.getdbt.com/docs/package-management) for more information on installing packages.
 ```yaml
 packages:
   - package: fivetran/netsuite
-    version: [">=0.5.0", "<0.6.0"]
+    version: [">=0.6.0", "<0.7.0"]
 ```
-
-## Configuration
-By default, this package looks for your netsuite data in the `netsuite` schema of your [target database](https://docs.getdbt.com/docs/running-a-dbt-project/using-the-command-line-interface/configure-your-profile). 
-If this is not where your netsuite data is, add the below configuration to your `dbt_project.yml` file.
-
+## Step 3: Define Netsuite.com or Netsuite2 Source
+As of April 2022 Fivetran made available a new Netsuite connector which leverages the Netsuite2 endpoint opposed to the original Netsuite.com endpoint. This package is designed to run for either or, not both. By default the `netsuite_netsuite_data_model` variable for this package is set to the original `netsuite` value which runs the netsuite.com version of the package. If you would like to run the package on Netsuite2 data, you may adjust the `netsuite_data_model` variable to run the `netsuite2` version of the package.
 ```yml
-# dbt_project.yml
+vars:
+    netsuite_netsuite_data_model: netsuite2 #netsuite by default
+```
 
-...
-config-version: 2
+## Step 4: Define database and schema variables
+By default, this package runs using your destination and the `netsuite` schema. If this is not where your Netsuite data is (for example, if your netsuite schema is named `netsuite_fivetran`), add the following configuration to your root `dbt_project.yml` file:
 
+```yml
 vars:
-    netsuite_database: your_database_name
-    netsuite_schema: your_schema_name
+    netsuite_database: your_destination_name
+    netsuite_schema: your_schema_name 
 ```
-This package allows users to add additional columns to the final model tables. 
-Columns passed through must be present in the downstream source tables. See 
-below for an example of how the passthrough columns should be configured within your `dbt_project.yml` file.
 
-```yml
-# dbt_project.yml
+## (Optional) Step 5: Additional configurations
 
-...
+### Passing Through Additional Fields
+This package includes all source columns defined in the macros folder. To add additional columns to this package, do so by adding our pass-through column variables to your `dbt_project.yml` file:
```

Comment:
should we include the more detailed changelog instructions?

Reply:
For sure! Just updated.